ledger/backend-api/src/main/scala/com/digitalasset/ledger/backend/api/v1/LedgerBackend.scala

```diff
@@ -65,6 +65,7 @@ import scala.concurrent.Future
   * the newly committed transaction.
   *
   */
+//TODO: do we need this trait? and the whole module?
 trait LedgerBackend extends AutoCloseable {
```

Comment:
turns out, that some of the data structures are still used from this module. I'd address that in a follow-up PR.